routers/api/v1/api.go

```diff
@@ -665,6 +665,7 @@ func RegisterRoutes(m *macaron.Macaron) {
 					m.Get("", repo.ListBranches)
 					m.Get("/*", context.RepoRefByType(context.RepoRefBranch), repo.GetBranch)
 					m.Delete("/*", reqRepoWriter(models.UnitTypeCode), context.RepoRefByType(context.RepoRefBranch), repo.DeleteBranch)
+					m.Post("", bind(api.CreateBranchRepoOption{}), repo.CreateBranch)
```

Comment:
reqRepoWriter(models.UnitTypeCode) missing

Reply:
Added it.